gensim/models/keyedvectors.py

```diff
@@ -339,7 +397,7 @@ def get_vector(self, word):
 
     def words_closer_than(self, w1, w2):
         """
-        Returns all words that are closer to `w1` than `w2` is to `w1`.
+        Return all words that are closer to `w1` than `w2` is to `w1`.
```

Comment:
Maybe better to use `Get` instead of `Return` here and everywhere in method description?